src/main/java/ai/lucidtech/las/sdk/Credentials.java

```diff
@@ -3,12 +3,20 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.Properties;
+import java.util.*;
+import java.time.Instant;
+import java.net.*;
+import org.json.*;
 
 
 public class Credentials {
-    private String accessKeyId;
-    private String secretAccessKey;
+    private String clientId;
+    private String clientSecret;
     private String apiKey;
+    private String authEndpoint;
+    private String apiEndpoint;
```

Comment:
Can these be "final"?

Reply:
They can, but what's the point in it?

Reply:
I don't see the need to allow modification of the variables after they have been defined in the constructor

Reply:
Can we refactor it later without breaking code for existing users? If yes, then I suggest skipping it for now

Reply:
Agreed.